example/comparison.py

```diff
@@ -5,9 +5,7 @@
 from __future__ import print_function
 import timeit
 import numpy as np
-from scipy.sparse import csr_matrix
 from scipy.sparse import rand
-from sparse_dot_topn import awesome_cossim_topn
```

Comment:
probably you have removed it because flake8 complained? It is actually used within timeit as a string.

Reply:
right ... fixed.